releasenotes/notes/fix_11143-d32a262538873a9d.yaml

```diff
@@ -0,0 +1,5 @@
+deprecations:
+  - |
+    The functions :func:`.num_to_latex_ket` and :func:`.numbers_to_latex_terms` 
+    have been removed. For similar functionality, see sympy's 
+    ``nsimplify`` and ``latex`` functions.
```

Comment:
```suggestion
upgrade:
  - |
    The functions :func: `qiskit.visualization.state_visualization.num_to_latex_ket()` and 
    :func: `qiskit.visualization.state_visualization.numbers_to_latex_terms()` are deprecated.
    For similar functionality, see sympy's  `nsimplify` and `latex` functions.

Reply:
@border-b You can not apply tag to PR, only maintainers can do that. (refer: [github docs](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels))